### DISCUSSION TITLE
Support for Languages

### DISCUSSION THREAD

**Discussion by ChakshuGautam** [beckn/DSEP-Specification#96] [Ideas]
## Use Cases
1. Languages for Course
2. Language for a Course Transcript

Currently, this is mapped as a tag. Also, this is not part of the fulfillment and comes as part of the item package as the learner has the choice to change both of these in the middle of the lecture.

## Proposed Solution
Most searches come for courses in a specific language. The [Descriptor](https://github.com/beckn/DSEP-Specification/blob/ee6832a26ff07a21e03ef3719fce6c43bc77d6e0/api/dsep.yaml#L1705) should be extended to include a separate category for `Language` in the following format.
```yaml
Descriptor:
      description: Describes the description of a real-world object.
      type: object
      properties:
        name:
          type: string
        code:
          type: string
        symbol:
          type: string
        short_desc:
          type: string
        long_desc:
          type: string
        images:
          type: array
          items:
            $ref: '#/components/schemas/Image'
        audio:
          type: string
          format: uri
        3d_render:
          type: string
          format: uri
        language:
          type: array
          format: object
            properties:
              language:
                $ref: '#/components/schemas/Language'
              tags:
                $ref: '#/components/schemas/Tags'
```

PR here - #94

**Comment by emmayank** [comment]
Hi @ChakshuGautam , 

Can you please help me in understanding why we should not put language under the tags present in the Item Schema. 

Thanks
Mayank Bansal
